src/Transport/Topology/Topologies/ForwardingTopology.cs

```diff
@@ -109,6 +110,14 @@ public Func<IDispatchMessages> GetDispatcherFactory()
             return () => container.Resolve<IDispatchMessages>();
         }
 
+        public StartupCheckResult ApplyPreStartupChecks()
+        {
+            var checks = container.ResolveAll<IStartupCheck>();
+            var compositeStartupChecker = new CompositeStartupCheck(checks);
```

Comment:
Imo this doesn't need to be configurable by the user, we can do this without using a container.